src/sentry/static/sentry/app/views/settings/incidentRules/presets.tsx

```diff
@@ -247,8 +247,19 @@ export function makeDefaultCta({
   incident,
   stats,
 }: PresetCtaOpts): PresetCta {
+  if (!incident) {
+    return {
+      buttonText: t('Open in Discover'),
+      to: '',
+    };
+  }
+
+  const extraQueryParams = {
+    display: 'top5',
```

Comment:
can we import and use the constant https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/discover/types.tsx#L8

Reply:
Makes sense, I will use it for all instances to be consistent